### PR TITLE
Update Irpachuza

### PR DESCRIPTION
**Pull request by WarriorGallade**

### CODE REVIEW

File: data/mods/gen9ssb/moves.ts [hunk 1]
```diff
@@ -1611,12 +1611,25 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 				msg = "We asked an AI to make a randbats set. YOU WON'T BELIEVE WHAT IT CAME UP WITH N.N";
 				break;
 			}
-			// TODO: ban mons with custom stats
-			const team = Teams.generate(randFormat, {name: target.side.name});
-			this.addMove('-anim', target, 'Wish', target);
-			// @ts-ignore set wants a sig but randbats sets don't have one
-			changeSet(this, target, team[0], true);
-			this.add(`c:|${getName((source.illusion || source).name)}|${msg}`);
+			while (true) {
```
Comment (HisuianZoroark) [inline data/mods/gen9ssb/moves.ts:1614]: infinite loop!

**Review by HisuianZoroark — COMMENTED** [review]
I don't think this is the proper way to do it, and I would rather take it over because I don't know how prone this check is to crashing.

File: data/mods/gen9ssb/moves.ts (hunk 1 above)
Comment (WarriorGallade) [reply data/mods/gen9ssb/moves.ts:1614]: it's _technically_ not infinite since there's a break at the bottom reached after a successful pokemon is found.

Comment (WarriorGallade) [reply data/mods/gen9ssb/moves.ts:1614]: anyway if you'd prefer to take this, go for it

**Comment by HisuianZoroark** [comment]
Fixed in 1449e0c39bcc400259285b528e51a8111a0444a3

**Pull request closed without merging**